test/unit/Persona/TokensTest.php

```diff
@@ -22,9 +22,11 @@ class TokensTest extends TestBase
     public function setUp()
     {
         parent::setUp();
-        $this->_wrongPrivateKey = file_get_contents('../keys/wrong_private_key.pem');
-        $this->privateKey = file_get_contents('../keys/private_key.pem');
-        $this->publicKey = file_get_contents('../keys/public_key.pem');
+
+        global $appRoot;
+        $this->_wrongPrivateKey = file_get_contents("{$appRoot}/test/keys/wrong_private_key.pem");
```

Comment:
running coverage from the test directory causes the path location to be wrong.

Reply:
I think I might have to look at doing something similar in depot rather than running coverage for each set of tests.